packages/components/src/spec-distributions.json

```diff
@@ -18,19 +18,19 @@
       "description": "The transform of the x scale",
       "value": 1.0,
       "bind": {
-        "input": "range",
-        "min": 0.1,
-        "max": 1
+        "input": "select",
+        "options": [0.1, 1],
+        "labels": ["log", "linear"]
       }
     },
     {
       "name": "yscale",
       "description": "The transform of the y scale",
       "value": 1.0,
       "bind": {
-        "input": "range",
-        "min": 0.1,
-        "max": 1
+        "input": "select",
+        "options": [0.1, 1],
+        "labels": ["log", "linear"]
```

Comment:
Do we really need to do this button in Vega? It seems to be uglier than it would be in regular React. Also, if there are just two elements, my guess is that a button would be nicer than a drop-down.

Reply:
Why is it uglier? Do you mean like, aesthetically (For which I agree, I might go ahead and style it)? Or code quality wise (for which I might disagree, I like the declarative style)?

I'm not sure about a button, I'll say that the buttons did confuse me the first time I used them. But a good compromise might be a checkbox? That people can tick for log scales?

Reply:
I mean aesthetically, sorry. I think a checkbox would be better than the current setup. I have a hunch that a one-button toggle would be even better, but it's more unusual (and likely harder to do via vega)

Reply:
It is harder to do in vega, and more unusual, and I think probably not as intuitive imo. I'll stick with checkboxes